src/Categories/Category/Construction/Kleisli/Symmetric.agda

```diff
@@ -0,0 +1,124 @@
+{-# OPTIONS --without-K --safe #-}
+module Categories.Category.Construction.Kleisli.Symmetric where
+
+open import Level
+open import Data.Product using (_,_)
+
+open import Categories.Category.Core using (Category)
+open import Categories.Monad using (Monad)
+open import Categories.Monad.Commutative using (CommutativeMonad)
+open import Categories.Monad.Commutative.Properties using (module CommutativeProperties)
+open import Categories.Category.Monoidal using (Monoidal)
+open import Categories.Category.Monoidal.Symmetric using (Symmetric)
+open import Categories.Category.Construction.Kleisli using (module TripleNotation)
+open import Categories.Category.Construction.Kleisli.Monoidal using (Kleisli-Monoidal)
+
+import Categories.Morphism.Reasoning as MR
+import Categories.Monad.Strong.Properties as StrongProps
+import Categories.Category.Monoidal.Symmetric.Properties as SymmProps
+import Categories.Category.Monoidal.Utilities as MonoidalUtils
+
+private
+  variable
+    o ℓ e : Level
+
+-- The Kleisli category of a commutative monad (where 𝒞 is symmetric) is symmetric.
+
+module _ {𝒞 : Category o ℓ e} {monoidal : Monoidal 𝒞} (symmetric : Symmetric monoidal) (CM : CommutativeMonad (Symmetric.braided symmetric)) where
+  open Category 𝒞
+  open MR 𝒞
+  open HomReasoning
+  open Equiv
+
+  open CommutativeMonad CM hiding (identityˡ)
+  open Monad M using (μ)
+  open TripleNotation M
+
+  open StrongProps.Left strength using (left-right-braiding-comm; right-left-braiding-comm)
+  open StrongProps.Left.Shorthands strength
+  open StrongProps.Right.Shorthands rightStrength
+  open MonoidalUtils.Shorthands monoidal using (α⇒; α⇐)
+
+
+  open Symmetric symmetric renaming (commutative to B-commutative)
+  open SymmProps symmetric
+  open CommutativeProperties braided CM
+
+  private
+    B : ∀ {X Y} → X ⊗₀ Y ⇒ Y ⊗₀ X
+    B {X} {Y} = braiding.⇒.η (X , Y)
+
+  Kleisli-Symmetric : Symmetric (Kleisli-Monoidal symmetric CM)
+  Kleisli-Symmetric = record 
+    { braided = record 
+      { braiding = record 
```

Comment:
Here too I think the braiding may be of independent interest and should be extracted.